exporter/exporter.go

```diff
@@ -347,77 +358,77 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 		txt  string
 		lbls []string
 	}{
-		"commands_duration_seconds_total":                 {txt: `Total amount of time in seconds spent per command`, lbls: []string{"cmd"}},
-		"commands_failed_calls_total":                     {txt: `Total number of errors prior command execution per command`, lbls: []string{"cmd"}},
-		"commands_rejected_calls_total":                   {txt: `Total number of errors within command execution per command`, lbls: []string{"cmd"}},
-		"commands_total":                                  {txt: `Total number of calls per command`, lbls: []string{"cmd"}},
-		"commands_latencies_usec":                         {txt: `A histogram of latencies per command`, lbls: []string{"cmd"}},
-		"latency_percentiles_usec":                        {txt: `A summary of latency percentile distribution per command`, lbls: []string{"cmd"}},
-		"config_client_output_buffer_limit_bytes":         {txt: `The configured buffer limits per class`, lbls: []string{"class", "limit"}},
```

Comment:
I haven't changed this part, but looks like `go fmt` wasn't run in for `config_client_output_buffer_limit_overcome_seconds` in https://github.com/oliver006/redis_exporter/pull/933

I think it's worth to add linter check for formatting in CI.

Reply:
Yeah, good catch, that wasn't working as expected.
I just pushed https://github.com/oliver006/redis_exporter/commit/827e6b5bfb6bbbee1fac3ae3d3b70da0489b9d1b to master - that should fix it.